src/MTGCore/Controllers/DeckController.cs

```diff
@@ -30,9 +31,41 @@ public DeckController(IRepoContext context, IMapper mapper, MTGService mtgservic
             _context = context;
             _userManager = userManager;
             _mapper = mapper;
-            
+
+        }
+
+        [HttpGet]
+        public IActionResult View()
+        {
+            var list = _context.DeckCards.Include(x => x.Card).Include(x => x.Deck).Where(x => x.DeckID == 1).ToList();
+            var results = list.GroupBy(g => g.CardID).ToList();
+
+            List<CardAmt> cardAmt = new List<CardAmt>();
+
+            foreach (var group in results)
+            {
+                CardAmt ca1 = new CardAmt();
+                CardDto card = new CardDto();
+                
+                var groupKey = group.Key;
+                ca1.Amount = group.Count();
+
+                foreach (var item in group)
```

Comment:
replace this with linq statement. Not actually using item why have i done this????